### PR TITLE
docs: add Layer0 deployment guide

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
## Check List

- [x] Others (Update, fix, translation, etc...)
  - Languages:
  - [x] `en` English

### CODE REVIEW

**Comment by tomap** [comment]
Hello, thank you for the suggestion, but a real and useful "how to" with detailed steps would be better than a marketing speech :)

**Comment by rishi-raj-jain** [comment]
Thanks for the feedback @tomap! 

I've updated the guide to include the steps involved with a deploy button, and hope now it doesn't sound like a marketing speech as I've now highlighted the uses of the platform which is it being a deployment platform, and their own EdgeJS and prefetching, and removed the marketing portion.

File: source/docs/one-command-deployment.md [hunk 1]
```diff
@@ -314,6 +314,46 @@ After a few moments, your website will be deployed.
...
+      maxAgeSeconds: 0,
+      serviceWorkerSeconds: 60 * 60 * 24,
+    },
+  })
+})
+```
+
+3. Deploy
+```bash
+npm run build
```
Comment (SukkaW) [inline source/docs/one-command-deployment.md:350]: Hexo doesn't use npm run build directly. So what does `npm run build` here for?

Comment (rishi-raj-jain) [reply source/docs/one-command-deployment.md]: Thanks for pointing that out @SukkaW, was referencing to the [demo](https://github.com/layer0-docs/layer0-hexo-example/blob/main/package.json) while I forgot to include the example. Updated to run `hexo generate` instead.

**Review by tomap — APPROVED** [review]

**Review by SukkaW — APPROVED** [review]

**Pull request merged**